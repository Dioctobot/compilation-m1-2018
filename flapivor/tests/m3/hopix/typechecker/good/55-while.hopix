val x : Ref<Int> = ref (4)

val main : Int =
    while (!x >? 0) {
      x := !x - 1
    };
    !x