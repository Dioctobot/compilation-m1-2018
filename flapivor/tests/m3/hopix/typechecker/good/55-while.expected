x : Ref<Int>
main : Int
